docs/ADMIN_GUIDE.ru.md

```diff
@@ -484,6 +484,43 @@ VirtualDisk            default              debian-10-root
 VirtualImage           default              ubuntu-2204
 ```
 
+### Мониторинг наличия образов
+
+Платформа может периодически проверять, что образы, хранящиеся в DVCR, по-прежнему существуют. Это помогает обнаружить ситуации, когда образы удаляются из DVCR (вручную или из-за проблем с хранилищем), в то время как соответствующие ресурсы VirtualImage или ClusterVirtualImage всё ещё показывают фазу `Ready`.
+
+При включении система выполняет легковесные запросы к DVCR для проверки существования манифестов образов. Если образ отсутствует, фаза ресурса меняется на `ImageLost`.
+
+Для настройки расписания мониторинга используйте параметр `.spec.settings.dvcr.imageMonitorSchedule`:
+
+```yaml
+apiVersion: deckhouse.io/v1alpha1
+kind: ModuleConfig
+metadata:
+  name: virtualization
+spec:
+  # ...
+  settings:
+    dvcr:
+      imageMonitorSchedule: "0 * * * *"
```

Comment:
I suggest not to give this setting in the module, but to set the minimum default value.